rustdoc: remove unused CSS `.out-of-band { font-weight: normal }`

notriddle opened the pull request:
This CSS was added in 083c3952e0d5473cd5c41a9eb7b4ffca18cc8e5f to normalize the appearance of out-of-band elements that were nested directly below headers.

Now, the only use of `out-of-band` is in the main page header, and it is nested below a wrapper, not the `<h1>` itself.

rust-highfive commented:
Thanks for the pull request, and welcome! The Rust team is excited to review your changes, and you should hear from @CraftSpider (or someone else) soon.

Please see [the contribution instructions](https://rustc-dev-guide.rust-lang.org/contributing.html) for more information.

notriddle committed:
rustdoc: remove unused CSS `.out-of-band { font-weight: normal }`

This CSS was added in 083c3952e0d5473cd5c41a9eb7b4ffca18cc8e5f to
normalize the appearance of out-of-band elements that were nested directly
below headers.

Now, the only use of `out-of-band` is in the main page header, and it is
nested below a wrapper, not the `<h1>` itself.

GuillaumeGomez commented:
Thanks! r=me once CI pass.

bors commented:
:pushpin: Commit 1adb4c338ab1b3bc2b44b054444b7ce8252241f6 has been approved by `notriddle`

It is now in the [queue](https://bors.rust-lang.org/homu/queue/rust) for this repository.

<!-- @bors r=notriddle 1adb4c338ab1b3bc2b44b054444b7ce8252241f6 -->
<!-- homu: {"type":"Approved","sha":"1adb4c338ab1b3bc2b44b054444b7ce8252241f6","approver":"notriddle","queue":"https://bors.rust-lang.org/homu/queue/rust"} -->

bors commented:
:pushpin: Commit 1adb4c338ab1b3bc2b44b054444b7ce8252241f6 has been approved by `GuillaumeGomez`

It is now in the [queue](https://bors.rust-lang.org/homu/queue/rust) for this repository.

<!-- @bors r=GuillaumeGomez 1adb4c338ab1b3bc2b44b054444b7ce8252241f6 -->
<!-- homu: {"type":"Approved","sha":"1adb4c338ab1b3bc2b44b054444b7ce8252241f6","approver":"GuillaumeGomez","queue":"https://bors.rust-lang.org/homu/queue/rust"} -->
